client/src/components/CheckBox.js

```diff
@@ -4,6 +4,11 @@ import { IconCheckmark } from "../icons/icons.js";
 import "./CheckBox.scss";
 
 export class CheckBox extends React.Component {
+  handleOnClick = (onClick, disabled, id) => {
+    if (!disabled && onClick) {
```

Comment:
Hmm... I did a little more reading and it apparently `.bind()` actually [creates a new function anyways](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind)

I think Alex is right in that it'd be best to access props inside this `handleOnClick`function

Reply:
You usually don't bind inline in the render, you'd do it in the constructor. Arrow function methods shouldn't need a `.bind`  though?

Reply:
I was doing it inline to pass in parameters. I was too focused on trying to pass the parameter, and should've known that binding in render creates a new function every render (doing it in constructor would not) But yeah Alex is right because without binding and using props straight in the function, it'll be auto-bound function